sql/create_user_table.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE user (
+  id INT PRIMARY KEY AUTO_INCREMENT,
+  name VARCHAR(32) NOT NULL,
+  email VARCHAR(32) NOT NULL,
+  password VARCHAR(255) NOT NULL,
+  roles VARCHAR(255)
```

Comment:
User could have multiple roles. change the implementation and sql as well.

Reply:
name -> firstname and lastname and email willl be used as usename fields and this will be unique

Reply:
Updated code changes; completed User have multiple roles   .
Added 3 roles  ROLE_ADMIN  ,ROLE_USER  and ROLE_INVENTORYMANAGER.